app/main.py

```diff
@@ -1,6 +1,18 @@
-from typing import Callable
+import functools
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    result = {}
+
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs) -> Any:
+        key = (func.__name__, args, frozenset(kwargs))
+        if key in result:
+            print("Getting from cache")
```

Comment:
Simplify by returning the value in the if statement and eliminating the else statement.